xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -5655,6 +5659,11 @@ platform.index.spaceIndex=Space Index
 platform.index.spaceIndexDescription=Pages in the {0} space:
 platform.index.spaceIndexDocumentListCreate=Create a new page
 
+#######################################
+## until 16.8.0RC1
```

Comment:
Should 16.10.0RC1 now.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3155/commits/0598ab85bd71030b48e172aef3540d630d8ee95c :+1: 
I checked the PR diff quickly and this should be the only version number that needed to be updated